strings/upper.py

```diff
@@ -13,7 +13,7 @@ def upper(word: str) -> str:
     """
 
     # converting to ascii value int value and checking to see if char is a lower letter
-    # if it is a capital letter it is getting shift by 32 which makes it a capital case
+    # if it is a lower letter it is getting shift by 32 which makes it a capital case
```

Comment:
```suggestion
    # if it is a lowercase letter it is getting shift by 32 which makes it a capital case
```
grammatical fix ('lower is not a good word to explain')